service/history/hsm/tree.go

```diff
@@ -126,7 +132,11 @@ type Node struct {
 // NewRoot creates a new root [Node].
 // Children may be provided from persistence to rehydrate the tree.
 // Returns [ErrNotRegistered] if the key's type is not registered in the given registry or serialization errors.
-func NewRoot(registry *Registry, t string, data any, children map[string]*persistencespb.StateMachineMap, backend NodeBackend) (*Node, error) {
+func NewRoot(registry *Registry,
```

Comment:
nit: I prefer to either have a comma and newline after the last arg or just keep the args in one line.

Reply:
Done